### DISCUSSION TITLE
Can you add on_member_kick, on_member_timeout?

### DISCUSSION THREAD

**Discussion by s22461-pjatk** [Pycord-Development/pycord#678] [Q&A] [answered]
Can you add on_member_kick and on_member_timeout?

I also have a question, is it possible to get who banned the user using the on_member_ban event? Because on_member_ban only gives discord.Guild and discord.User

**Comment by Lulalaby** [comment, marked as answer]
Not possible due to discord limitations.

Use these alternatives (Don't know the exact names rn, search the docs):
- on_member_kick > member remove
- on_member_timeout > member update

For on_member_ban fetch the audit log.

**Reply by s22461-pjatk** [reply]
Thx
